event-subscription/service/example-subscription.yaml

```diff
@@ -5,13 +5,10 @@ metadata:
   labels:
     example: event-bus
 spec:
-  endpoint: http://event-email-service.<environment>:3000/v1/events/register
+  endpoint: http://event-email-service.workshop:3000/v1/events/register
```

Comment:
why did you fixate the namespace? I believe it should stay optional for the deployer/user and the guide already prompt the user to set the desired namespace as well as use to `$sed` the file to set the namespace in this file.